### DISCUSSION TITLE
components that are inheriting from AbpCrudPageBase

### DISCUSSION THREAD

**Discussion by 274188A** [abpframework/abp#6224] [General] [answered]
I'm have components that are inheriting from AbpCrudPageBase (not the new AbpComponentBase) because the component is just pure CRUD.

So I could end up with a AbpCrudPageBase page having a component of type AbpCrudPageBase - hopefully this is OK?

**Comment by hikalkan** [comment, marked as answer]
Sure, that's OK. [The tutorial](https://docs.abp.io/en/abp/4.0/Tutorials/Part-1?UI=Blazor) also does like that. `AbpCrudPageBase` already inherits from the `AbpComponentBase`.
